docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md

```diff
@@ -215,7 +215,7 @@ Ensure that you have DHCP running and run the create command again.
 If there are any IPv4 IPs assigned, check if one of the VMs have the controlPlane IP specified in `Cluster.spec.controlPlaneConfiguration.endpoint.host` in the clusterconfig yaml.
 If this IP is not present on any control plane VM, make sure the `network` has access to the following endpoints:
 
-{{% content "domains.md" %}}
+{{% content "../../reference/vsphere/domains.md" %}}
```

Comment:
Does this work? I found out that I needed to change the content shortcode to use `Page` instead of `Site`

Reply:
Well, it stopped complaining, but I noticed it doesn't render right

Reply:
fixed